site_settings/models.py

```diff
@@ -20,5 +22,41 @@ class FooterSettings(BaseSetting):
             FieldPanel("twitter"),
             FieldPanel("instagram"),
             FieldPanel("address"),
-        ], heading="Footer Settings")
+        ],
+                        heading="Footer Settings")
+    ]
+
+
+@register_setting
+class AlertBanner(BaseSetting):
+    """Create and toggle a site-wide alert banner."""
+
+    INFO = 'alert-info'
+    LOW = 'alert-low'
+    HIGH = 'alert-high'
+    ALERT_TYPES = (
+        (INFO, 'Informational Alert'),
+        (LOW, 'General Alert'),
+        (HIGH, 'Critical Alert'),
+    )
+
+    enable = models.BooleanField(
+        default=False,
+        help_text='Checking this box will enable the alert on all web pages.')
+    alert_message = RichTextField(blank=True,
+                                  null=True,
+                                  help_text="Alert Message")
+    alert_level = models.CharField(
+        max_length=25,
+        choices=ALERT_TYPES,
+        default=INFO,
+    )
+
+    panels = [
+        MultiFieldPanel([
+            FieldPanel("enable"),
+            FieldPanel("alert_message"),
+            FieldPanel("alert_level"),
+        ],
+                        heading="Alert Banner")
```

Comment:
There's some weird stuff going on in this file with linting. Since we haven't set that up for this site, I'm not going to have it be a blocker, however, this line `61` looks really off to me. `heading` should line up with `MultiFieldPanel`. I imagine it's not causing any problems because it's not blowing up but we might want to setup linting sometime in the future for this site. Just a suggestion.

Reply:
Yeah, that is wild. I thought we added a linter to this project, but I must be wrong.